jekyll/_cci2/executor-types.md

```diff
@@ -104,25 +109,20 @@ set the [`machine` key]({{ site.baseurl }}/2.0/configuration-reference/#machine)
 version: 2
 jobs:
   build:
-    machine: true
+    machine:
+      image: ubuntu-1604:201903-01    # pins image to specific version
 ```
 
-The default image for the machine executor is `circleci/classic:latest`.
+The default image for the machine executor is `circleci/classic:latest`.  If you don't specify an image, jobs will run on the default image - which is currently circleci/classic:201710-01 but may change in future.
```

Comment:
Could we drop any mention of `latest` on this page? It's not really the latest, and the default is really `201710-01`, which happened to also be `latest` last time we updated that tag 😢